### ISSUE TITLE
CORS policy error in signin page

### ISSUE THREAD

**Issue by diabloexodia** [eevan7a9/real-estate-management#27]
signin:1 Access to XMLHttpRequest at 'https://rem-db.glitch.me/auth/signin' from origin 'https://real-estate-management.netlify.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.

**Comment by leo-1228** [comment]
Yes, I can fix this, give me assign. If you want to discuss with me, I am always on telegram. this is my telegram @leo1228codeforce
contact me.
